infra/nightly-resources/index.js

```diff
@@ -126,34 +132,62 @@ function groupByBenchmark(benchList) {
     return groupedBy;
 }
 
+// Outputs current_number - baseline_number in a human-readable format
+// If baseline_number is undefined, it will return N/A
+function getDifference(current, baseline) {
+    // if b is undefined, return a
+    if (baseline === undefined) {
+        return { class: "max", value: "N/A" };
+    } else {
+        var difference = current - baseline;
+        // if the difference is negative it will already have a "-"
+        var sign = difference < 0 ? "" : "+";
+        var color = "";
+        if (difference < 0) {
+            color = "min";
+        } else if (difference > 0) {
+            color = "max";
```

Comment:
nit: should change these class names to `good` / `bad` or something now that they're not always just for mins/maxes